tempesta_fw/http.c

```diff
@@ -391,6 +407,7 @@ tfw_http_prep_redirect(TfwHttpMsg *resp, unsigned short status, TfwStr *cookie,
 	static TfwStr c_len_crlf = {
 		.ptr = S_REDIR_C_LEN, .len = SLEN(S_REDIR_C_LEN) };
 	TfwStr host, *rh, *cookie_crlf = &crlf, *r_end;
```

Comment:
Not an issue of this PR, but there is one issue more. According to [RFC 7230](https://tools.ietf.org/html/rfc7230#section-5.4):
>  When a proxy receives a request with an absolute-form of
   request-target, the proxy MUST ignore the received Host header field
   (if any) and instead replace it with the host information of the
   request-target.  

That means `req->host` must take precedence over `&req->h_tbl->tbl[TFW_HTTP_HDR_HOST]`, but in a few lines below we use 'Host :' header as primary value and `req->host` as fallback value.

Reply:
Corrected.